features/upgrade/machines/machines.feature

```diff
@@ -75,3 +75,66 @@ Feature: Machine-api components upgrade tests
     When I scale the machineset to -1
     Then the step should succeed
     And the machineset should have expected number of running machines
+
+  @upgrade-prepare
+  @admin
+  Scenario: Cluster should automatically scale up and scale down with clusterautoscaler deployed - prepare
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+    And I use the "openshift-machine-api" project
+
+    Given I obtain test data file "cloud/cluster-autoscaler.yml"
+    When I run the :create admin command with:
+      | f | cluster-autoscaler.yml |
+    Then the step should succeed
+    And 1 pods become ready with labels:
+      | cluster-autoscaler=default,k8s-app=cluster-autoscaler |
+
+  # @author jhou@redhat.com
+  # @case_id OCP-30783
+  @upgrade-check
+  @admin
+  @destructive
+  Scenario: Cluster should automatically scale up and scale down with clusterautoscaler deployed
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+    And I use the "openshift-machine-api" project
+    And admin ensures machine number is restored after scenario
+
+    Given I clone a machineset and name it "machineset-clone-28108"
```

Comment:
What's the meaning of 28108 ?

Reply:
The case id, apparently it should be 30783 here. Updated, thanks